### DISCUSSION TITLE
Use trivy as go library

### DISCUSSION THREAD

**Discussion by DmitryZ-outten** [aquasecurity/trivy#987] [Q&A]
It would be great to have a simple example on how to use Trivy in go as a library to scan docker image. 
There is already a similar project: https://github.com/developer-guy/image-scanning-using-trivy-as-go-library
But it would be great to have this in the aquasecurity/trivy project itself.

**Comment by allanchua101** [comment]
I would love to see one for NodeJS as well 👍
